src/main/resources/view/PersonListCard.fxml

```diff
@@ -29,8 +29,8 @@
       </HBox>
       <FlowPane fx:id="tags" />
       <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+      <Label fx:id="matriculationNumber" styleClass="cell_small_label" text="\$email" />
```

Comment:
Shouldn't the text name for this be `matriculationNumber`?

Reply:
ahhh I see